qualtran/bloqs/qft/qft_phase_gradient.py

```diff
@@ -75,3 +74,14 @@ def decompose_from_registers(
         if self.with_reverse:
             for i in range(self.bitsize // 2):
                 yield cirq.SWAP(q[i], q[-i - 1])
+
+
+@bloq_example
+def _qft_phase_gradient_small() -> QFTPhaseGradient:
+    qft_phase_gradient_small = QFTPhaseGradient(3)
+    return qft_phase_gradient_small
+
```

Comment:
Ditto: Add symbolic example and add bloq_autotester tests for both the examples.